### DISCUSSION TITLE
English version

### DISCUSSION THREAD

**Discussion by haimat** [PFCCLab/PPOCRLabel#111] [General]
Hello, is there an English version of PPOCRLabel?

**Comment by haimat** [comment]
Ahh, found it - just star the app with `--lang en` command line paramter.

**Comment by GreatV** [comment]
Yes, there is an English version of PPOCRLabel available. The tool supports multiple languages, including English, and allows users to annotate OCR datasets. You can use the English interface by running the PPOCRLabel tool without any additional configuration or by specifying the language parameter as English.

To run PPOCRLabel in English:

1. Install PPOCRLabel using `pip install PPOCRLabel`.
2. Start the application using the following command:
   ```bash
   PPOCRLabel  # Default mode for detection and recognition in English
   ```

Additionally, the English version supports features such as:
- Text annotation
- Table annotation
- Irregular text annotation
- Key information annotation

For more details and recent updates, you can refer to the [English documentation](https://github.com/PFCCLab/PPOCRLabel/discussions/111).

---
*Response generated by [feifei-bot](https://github.com/GreatV/feifei) | chatgpt-4o-latest*
